azure-mgmt-eventhub/src/test/java/com/microsoft/azure/management/eventhub/EventHubTests.java

```diff
@@ -342,27 +342,27 @@ public void canConfigureEventHubDataCapturing() throws Exception {
 
         Creatable<StorageAccount> storageAccountCreatable = storageManager.storageAccounts()
                 .define(stgName)
-                .withRegion(REGION)
-                .withNewResourceGroup(RG_NAME)
-                .withSku(StorageAccountSkuType.STANDARD_LRS);
+                    .withRegion(REGION)
+                    .withNewResourceGroup(RG_NAME)
+                    .withSku(StorageAccountSkuType.STANDARD_LRS);
 
         Creatable<EventHubNamespace> namespaceCreatable = eventHubManager.namespaces()
                 .define(namespaceName)
-                .withRegion(REGION)
-                .withNewResourceGroup(RG_NAME);
+                    .withRegion(REGION)
+                    .withNewResourceGroup(RG_NAME);
 
         final String containerName1 = "eventsctr1";
 
         EventHub eventHub1 = eventHubManager.eventHubs()
                 .define(eventHubName1)
-                .withNewNamespace(namespaceCreatable)
-                .withNewStorageAccountForCapturedData(storageAccountCreatable, containerName1)
-                .withDataCaptureEnabled()
-                // Window config is optional if not set service will choose default for it2
-                //
-                .withDataCaptureWindowSizeInSeconds(120)
-                .withDataCaptureWindowSizeInMB(300)
-                .create();
+                    .withNewNamespace(namespaceCreatable)
+                    .withNewStorageAccountForCapturedData(storageAccountCreatable, containerName1)
```

Comment:
We might need a better name for this method, something like .withDataCapturingContainer() may be?